### ISSUE TITLE
Man pages

### ISSUE THREAD

**Issue by travankor** [GrayJack/coreutils#116]
`coreutils` is not really a first-class *NIX citizen without man pages.

There's several different tools to solve this with, idk what will work best. The NetBSD and OpenBSD man pages are excellent reference material and can be partially copied. I would avoid GNU man pages (and texinfo) at all costs.

**Comment by GrayJack** [comment]
I agree!! Looks like clap v3 will have a feature to produce man pages, that was the reason that I picked clap to use universally in the project

Here a [reference](https://rust-cli.github.io/book/in-depth/docs.html) where I got it
